### ISSUE TITLE
Enhance usability for 'dump trace --keys  and --keysfile' option

### ISSUE THREAD

**Issue by samuelbroscheit** [uma-pi1/kge#75]
Documentation for the 'dump trace --keys  and --keysfile' option are needed. Its not clear how to use them. 

Also helpful for this would be a 'dump trace --list_keys' option.

**Comment by rgemulla** [comment]
Semantics is either "key" or "name=key", where key is a key in the trace and name is the name of the columns in the CSV. Assigned to Patrick to add documentation.

**Comment by Nzteb** [comment]
Yep. "Key" can be everything that either exists in the trace as key or in the config as key, which can be different for different models etc (it creates an empty field in the csv if not existing) There are also special keys as $folder $machine,.. , --list_keys might not know about all of them beforehand but could output the special keys and a subset of keys that are always applicable.

**Comment by samuelbroscheit** [comment]
Sorry my proposal for the `--list_keys` option should have stated that it should parse the trace and print out all potential keys in this trace. 

Example:

`kge dump trace <folder_name> --list_keys`

**Comment by rgemulla** [comment]
Almost addressed by #83. Only ``--list-keys`` is open.

**Comment by Nzteb** [comment]
To be consistent, --list-keys should also output the possible keys of the config as these can also be added to --keys and --keysfile.  Should it output all keys from the default config? I'd prefer to only output the minimal keys corresponding to 'dump config --minimal'  and maybe make a note that all the default keys in config_default.yaml can also be used.

**Comment by rgemulla** [comment]
Perhaps the output could be along the lines of:

```
Default keys: ...
Special keys: $folder, ...
Keys found in trace: <sorted list>
Keys found in config: <sorted list>
```

It's fine to simply list all config keys in alphabetical order.

**Comment by rgemulla** [comment]
Closed via #87
